### DISCUSSION TITLE
docker yaml files '4g-volts-deploy.yaml'

### DISCUSSION THREAD

**Discussion by easybodge** [herlesupreeth/docker_open5gs#392] [General]
Hi Just checking to make sure I understand it correctly. Is your project fully setup such that when we run the 4g-volte-deploy.yaml for the first time as '4g-volte-deployment up'after docker is installed, it will fetch and install all required containers? so we dont have to manually pull every docker image using the direct links? 
thanks

**Comment by herlesupreeth** [comment]
yes, your understanding is correct. Let me know if you face any issues
